src/StaticAnalyzers.cmake

```diff
@@ -2,11 +2,18 @@ macro(enable_cppcheck CPPCHECK_OPTIONS)
   find_program(CPPCHECK cppcheck)
   if(CPPCHECK)
 
+    if(CMAKE_GENERATOR MATCHES ".*Visual Studio.*")
```

Comment:
Shouldn't this be `${CMAKE_GENERATOR}`. I find CMake's rules for variables still confusing.